execution/evm/execution.go

```diff
@@ -287,6 +287,11 @@ func (c *EngineClient) getBlockInfo(ctx context.Context, height uint64) (common.
 	return header.Hash(), header.Root, header.GasLimit, header.Time, nil
 }
 
+// Rollback rolls back the state to the specified height.
+func (c *EngineClient) Rollback(ctx context.Context, height uint64) error {
```

Comment:
Should be done in a follow-up by someone with more context with the EVM execution environment. This is enough to unblock me for ev-abci rollback.